src/main/java/core/basesyntax/service/WriterServiceImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.PrintWriter;
+import java.util.Map;
+
+public class WriterServiceImpl implements WriterService {
```

Comment:
better create 2 separate services 
- to create report 
- to write String to file